### ISSUE TITLE
Anchor links within notebook sandbox pointing to cells don't works on Firefox

### ISSUE THREAD

**Issue by saneef** [observablehq/feedback#294]
**Describe the bug**
The anchor links within notebook sandbox pointing to cells don't work on Firefox (`v94.0.2`). Works fine on Safari and Chrome.

**To Reproduce**
1. Go to https://observablehq.com/d/9c9379d58e125b91
2. Click on `Go Down` or `Go Up` links

**Expected behavior**
The browser should navigate to the pointed cell (in the same page).

**Desktop (please complete the following information):**
 - OS: macOS 12.0.1
 - Browser: Firefox
 - Version: 94.0.2

**Comment by CobusT** [comment]
Strange. It works for me. Same version of Firefox and macOS.

**Comment by mootari** [comment]
Then it may be related to [Site Isolation](https://hacks.mozilla.org/2021/05/introducing-firefox-new-site-isolation-security-architecture/) which is rolled out over several weeks, according to the [release notes](https://www.mozilla.org/en-US/firefox/94.0/releasenotes/).

**Comment by saneef** [comment]
@mootari You are right, it is related to the Site Isolation. I tried after disabling the 'Site Isolation'; the anchor link works fine.

If anyone want to enable/disable the Site Isolation in Firefox: 

1. Go to the URL `about:config`
2. Search for `fission.autostart`
3. Toggle its value to `true` to enable, or `false` to disable the Site Isolation feature.
4. Restart Firefox

**Comment by CobusT** [comment]
Great. Looks like we can close this issue

**Comment by mootari** [comment]
@CobusT How so? If anything, the issue will become more relevant in the coming weeks, and may require a workaround in Observable's UI code.

**Comment by CobusT** [comment]
Oops. Reading this again, it looks like this will become more prevalent. Will reopen it so we can investigate further.

**Comment by saneef** [comment]
The problem is not just with anchor links. Links with URL pointing outside `observablehq.com` also don't work. You can try more links on the [same notebook](https://observablehq.com/d/9c9379d58e125b91).
